angelsindustries/data-final-fixes.lua

```diff
@@ -83,6 +83,8 @@ end
 
 --tech updates part 1
 require("prototypes.overrides.global-tech")
+--double check all items are updated if they had been tweaked wrong
+require("prototypes.overrides.components-angels-entity-update.components-angels-building-updates")
```

Comment:
I do think this is just a detail of the big mess.. I might make a PR to fix this a bit better instead... I approve as it is now, just putting this suggestion here still
```suggestion
require("prototypes.overrides.components-angels-entity-update.components-angels-building-final-fixes")
```

Reply:
good point, i have renamed the file, and the call to match

Reply:
I'm really not sure how to actually fix the main issues with this outside of either locking out AAII or adding our buildings to a cache then re-analyzed and processed right at the end. 
The other option is to ask Earendel to not change the building recipes if components mode is active.

Reply:
I think this is a good compromise for now...